src/current/_data/releases.yml

```diff
@@ -7460,4 +7460,31 @@
     docker_arm_experimental: false
     docker_arm_limited_access: false
   source: true
-  previous_release: v24.3.0-beta.3
\ No newline at end of file
+  previous_release: v24.3.0-beta.3
+
+- release_name: v24.3.1
+  major_version: v24.3
+  release_date: '2024-12-12'
+  release_type: Production
+  go_version: go1.22.8
+  sha: f9b40bb0eb7ed20acd68f24afc4c402614a9274b
+  has_sql_only: true
+  has_sha256sum: true
+  mac:
+    mac_arm: true
+    mac_arm_experimental: true
+    mac_arm_limited_access: false
+  windows: true
+  linux:
+    linux_arm: true
+    linux_arm_experimental: false
+    linux_arm_limited_access: false
+    linux_intel_fips: true
+    linux_arm_fips: false
+  docker:
+    docker_image: cockroachdb/cockroach
+    docker_arm: true
+    docker_arm_experimental: false
+    docker_arm_limited_access: false
+  source: true
+  previous_release: v24.3.0
```

Comment:
You need to add in the cloud-first part here. 

```
cloud_only: true
cloud_only_message_short: 'Available only for select CockroachDB Cloud clusters'
cloud_only_message: >
    This version is currently available only for select
    CockroachDB Cloud clusters. To request to upgrade
    a CockroachDB self-hosted cluster to this version,
    [contact support](https://support.cockroachlabs.com/hc/requests/new).
```

I actually forgot that in my PR too, so I super remembered reviewing yours!

Reply:
Thank you!!